.snippets/code/builders/interact/substrate-api/polkadot-js-api/balance-event.ts

```diff
@@ -1,3 +1,4 @@
+import '@polkadot/api-augment';
```

Comment:
I get tons of errors with this and tons without haha, so I don't know what's going on here - did you try this out?

Reply:
Hmmm, yes, I tested all the scripts copy/pasting from the article but probably I had something else installed that made it work.
I'll try again in using a fresh environment!

Reply:
ok thanks!